ci-operator/step-registry/openshift/e2e/aws/heterogeneous/conf/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
```

Comment:
If your approver-set for this step is expected to continue to match the parent directory, you can use a symlink here instead of the copy/paste (e.g. see [`hack/dedup-owners.sh`](https://github.com/openshift/release/blob/master/hack/dedup-owners.sh)).